app/src/main/java/ca/llamabagel/transpo/data/SearchRepository.kt

```diff
@@ -24,7 +24,7 @@ class SearchRepository @Inject constructor(private val database: TransitDatabase
 
         if (stops.isNotEmpty()) {
             searchResults.add(SearchResult.CategoryHeader("Stops"))
-            searchResults.addAll(stops.map { SearchResult.StopItem(it.name, it.code, "No upcoming trips") })
+            searchResults.addAll(stops.map { SearchResult.StopItem(it.name, "• ${it.code}", "No upcoming trips") })
```

Comment:
Later, we will opt for using "UiModel" classes that handle this type of appending at the model level. I have a PR coming up that will demo that.